designs/index.md

```diff
@@ -129,9 +138,9 @@ It is also possible to first iterate on a Google Doc, and then convert it to
 Markdown for posterity.
 
 
-## Using Google Docs
+### Using Google Docs
 
-Create a world-readable document on [Google Doc](https://docs.google.com). To
+Create a world-readable document on [Google Docs](https://docs.google.com). To
```

Comment:
I think it would be more helpful to start with the recommendation to use the template and then talk about how to make it world readable.

Reply:
Done.